modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -12,6 +12,43 @@ Each ACM Lightning component has different configuration options and adds differ
 
 Many ACM Lightning components show data from Anypoint Platform. ACM admins can change some of these data fields, such as API names, descriptions, and icons. ACM saves these changes but does not push the changes back to Anypoint Platform.
 
+== ACM Search Box [[acm-search-box]]
+
+The ACM Search Box component enables users to search both for content that exists in Community Cloud, such as knowledge articles, support cases, and announcements, and content that exists in Anypoint Platform, such as APIs and client applications. Results from the search are presented in the <<acm-search-results>> component.
+
+You can specify values for the following parameters of the ACM Search Box:
+
+Component Title:: Specifies the title for the search input field.
+Number of Results - Instant Incremental Response:: Specifies the number of results shown as the user types a query.
+Search Areas:: Selects which types of content to search. Choices include APIs, Applications, Knowledge Articles, Cases, Pages, and Forums.
+Linked Result Page:: Specifies the URL of the search results page shown after the user enters a search.
+
+=== Search Configuration
+
+Search is a basic expected feature in any developer portal, and becomes important as the number of APIs grows to around 50 and beyond.
+
+The example portals and default pages have one ACM Search Box and one ACM Search Results component. This is the best practice to minimize the chance of misconfigurations.
+
+The ACM Search Box parameter Linked Result Page specifies the search results page. That page must have an ACM Search Results component to display the search results.
+
+The ACM Search Results component shows only results of the content types specified in the Search Areas parameters of the ACM Search Box component. If a content type is not selected then the search results do not include any results of that content type.
```

Comment:
Shouldn't this text be moved to the following subtopic by the same name?
Also, GLOBALLY fully spell or use the accepted short form API Community Manager rather than ACM.
```suggestion
The API Community Manager Search Results component shows results of only the content types specified in the Search Areas parameters of the API Community Manager Search Box component. If a content type is not selected then the search results do not include any results of that content type.
```

Reply:
Rewrote this to clarify that it's focused on the Search Areas parameter.

Unfortunately the UI has a long alphabetical list of components, including the names "ACM Search Box" and "ACM Search Results", so if the docs call them "API Community Manager Search Box" and so on, we'll send the users to the wrong place in the list. I'll raise this with the PM next week when I scrub "ACM" from all pages in the docs, and hopefully we can get it fixed in the UI.